### PR TITLE
None docstrings fix

### PR DESCRIPTION
**Pull request by jcharlong**
## Type of changes

- [x] Bug fix
- [ ] New feature
- [ ] Documentation / docstrings
- [ ] Tests
- [ ] Other

## Checklist

- [x] I've run the latest [black](https://github.com/ambv/black) with default args on new code.
- [x] I've updated CHANGELOG.md and CONTRIBUTORS.md where appropriate.
- [ ] I've added tests for new code.
- [x] I accept that @willmcgugan may be pedantic in the code review.

## Description

Fixes an issue when docstrings are set to `None`. Issue #415.

### CODE REVIEW

**Comment by jcharlong** [comment]
Fixes issue #415 .

File: fs/base.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ def _method(*args, **kwargs):
 """.format(
         method.__name__
     )
-    if hasattr(_method, "__doc__"):
+    if hasattr(_method, "__doc__") and _method.__doc__ is not None:
```
Comment (lurch) [inline fs/base.py:87]: Is this the same as `if getattr(_method, "__doc__", None) is not None:` ?

File: fs/_version.py [hunk 2]
```diff
@@ -1,3 +1,3 @@
 """Version, used in module and setup.py.
 """
-__version__ = "2.4.11"
+__version__ = "2.4.12"
```
Comment (althonos) [inline fs/_version.py:3]: Can you remove the bump for the version number ? We only update it in the commit where we publish a new release.

**Review by althonos — CHANGES_REQUESTED** [review]
Good job, thanks! Just waiting for you to revert changes on the `fs._version` file and then I'll merge. Can you add yourself to the `CONTRIBUTORS.md` file as well if possible?

File: fs/_version.py (hunk 2 above)
Comment (jcharlong) [reply fs/_version.py]: Fixed.

**Comment by jcharlong** [comment]
Reverted the version bump & added myself to the contributors. :+1:

**Review by althonos — APPROVED** [review]

**Pull request merged**